Sources/Plasma/NucleusLib/pnModifier/CMakeLists.txt

```diff
@@ -28,7 +28,6 @@ target_link_libraries(
         pnNetCommon
     PRIVATE
         pnMessage
-        plNetMessage
```

Comment:
GitHub doesn't let me comment in the right place, but: you can now remove the extra PubUtilLib include directory a few lines below:

```cmake
target_include_directories(pnModifier PRIVATE "${PLASMA_SOURCE_ROOT}/PubUtilLib")
```

Reply:
ooh, good catch! Thanks